composer.json

```diff
@@ -975,10 +975,10 @@
         "spryker/docker-chromedriver": "dev-master",
         "spryker/testify": "^3.0.0",
         "spryker/web-profiler": "^1.5",
-        "spryker/web-profiler-extension": "^1.0.0",
-        "stecman/symfony-console-completion": "^0.8.0",
-        "symfony/var-dumper": "^5.2.8",
-        "symfony/web-profiler-bundle": "^4.0.0 || ^5.0.0"
+        "spryker/web-profiler-extension": "*",
+        "stecman/symfony-console-completion": "*",
+        "symfony/var-dumper": "*",
+        "symfony/web-profiler-bundle": "*"
```

Comment:
I would not unspecify those, as it makes it harder to track the reason and diff regarding nonsplit.
Is there any reason this needs to be made `*`?

Reply:
Those are dev dependencies. They were blocking us to update Symfony components which have a vulnerability. It was extremely time-consuming to find out why newer versions aren't downloaded.

Reply:
ok, for dev dependencies this is probably fair enough.

Reply:
Bear in mind, that setting wildcard can have unforeseen and usually bad consequences when using prefer lowest checks, as it will often go back too far and break.
So where we can avoid it we still should.